### ISSUE TITLE
how to speak in chinese

### ISSUE THREAD

**Issue by chinholim** [ndarilek/tts-rs#6]

**Comment by chinholim** [comment]
cannot speak in chinese

**Comment by ndarilek** [comment]
Yeah, language/voice-switching is missing at the moment. I personally don't need either feature right now, so would like to see it implemented by someone who *does* so I don't get the implementation wrong.

I'd like to see multilingual support integrated with voice enumeration/selection if it makes sense to do so. Maybe you can check out the currently unfinished voices PR for inspiration? Might only need a language/locale field on the voice if there isn't one already.

**Comment by piedoom** [comment]
Adding my +1 for this feature. Unfortunately I'm not confident in my ability to add this myself, but will try and get comfortable with the lib and may be able to see how trivial/difficult it is to add. Looking across other TTS crates on [lib.rs](https://lib.rs/search?q=text-to-speech) this appears to be the only (or of few) solution that does not require a web API which is extremely useful.
